test/functional/api/v2/organizations_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class Api::V2::OrganizationsControllerTest < ActionController::TestCase
+  def setup
+    @organization = taxonomies(:organization1)
+    Rabl.configuration.use_controller_name_as_json_root = false
```

Comment:
I don't think the rabl configuration should be changed inside a test, as it might leak out of this test and affect others, and it also doesn't match the real configuration.